### PR TITLE
fixes bug 1119378 - Upgrade to Django 1.6

### PR DESCRIPTION
**Pull request by peterbe**
@rhelmer r?

### CODE REVIEW

**Comment by peterbe** [comment]
I've tested it a bunch locally and it works fine. One big change was around the list type inputs for test driving the API and that works.

File: webapp-django/crashstats/api/views.py [hunk 1]
```diff
@@ -99,9 +99,9 @@ class APIWhitelistError(Exception):
 class MultipleStringField(forms.TypedMultipleChoiceField):
     """Field that do not validate if the field values are in self.choices"""
 
-    def to_python(self, value):
-        """Override checking method"""
-        return map(self.coerce, value)
+    # def to_python(self, value):
+    #     """Override checking method"""
+    #     return map(self.coerce, value)
```
Comment (rhelmer) [inline webapp-django/crashstats/api/views.py]: why is this commented out?

Comment (peterbe) [inline webapp-django/crashstats/api/views.py]: Because I'm sloppy. Will fix.

**Comment by rhelmer** [comment]
r+

**Pull request merged**